unitary/examples/quantum_chinese_chess/enums.py

```diff
@@ -42,26 +43,30 @@ class MoveVariant(enum.Enum):
 
 
 class Piece(enum.Enum):
-    EMPTY = '.'
-    SOLDIER = 's'
-    CANNON = 'c'
-    ROOK = 'r'
-    HORSE = 'h'
-    ELEPHANT = 'e'
-    ADVISOR = 'a'
-    GENERAL = 'g'
-    def type_of(c: str) -> Optional["Piece"]:
+    EMPTY = "."
```

Comment:
Thanks a lot for the fix! Just curious did you use a tooling to identify/suggest such formatting/typing fixes?

Reply:
Yeah, I used 'black' for formatting and 'mypy' for typing.  
I plan to add them to continuous integration once I fix up all the issues (otherwise they will just fail until we do).